dts/bindings/sensor/rohm,bh1750.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022, morsisko
+# SPDX-License-Identifier: Apache-2.0
+
+description: Rohm BH1750 ambient light sensor.
+
+compatible: "rohm,bh1750"
+
+include: i2c-device.yaml
+
+properties:
+    resolution:
+      type: int
+      required: false
+      default: 1
```

Comment:
Please read https://docs.zephyrproject.org/latest/guides/dts/bindings.html#default-values-for-properties carefully and add a justification for why this default was chosen to the property description or make it required.

Reply:
This is the default value that is stored inside the chip itself after power on, it is chosen by the sensor manufacturer. Not sure if it's good idea to make it required, because then user needs to dive into the chip manual and select the value on their own, but I believe for a lot of users the default settings are good enough. However if you think it's better idea to remove the deafault values i can do it.

Justification from link you've posted above:
Those values are: `defaults which match the vendor-specified power-on reset value (as long as they are independent from other properties)`